internal/cmd/report/sbom/sbom.go

```diff
@@ -0,0 +1,85 @@
+package sbom
+
+import (
+	"fmt"
+
+	"github.com/debricked/cli/internal/report"
+	"github.com/debricked/cli/internal/report/sbom"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var commitId string
+var repositoryId string
+var branch string
+var vulnerabilities bool
+var licenses bool
+var output string
+
+const CommitFlag = "commit"
+const RepositorylFlag = "repository"
+const TokenFlag = "token"
+const BranchFlag = "branch"
+const VulnerabilitiesFlag = "vulnerabilities"
+const LicensesFlag = "licenses"
+const OutputFlag = "output"
+
+func NewSBOMCmd(reporter report.IReporter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sbom",
+		Short: "Generate SBOM report",
+		Long: `Generate SBOM report for chosen commit and repository. 
+For an example of the SBOM format see https://github.com/debricked/blog-snippets/blob/main/example-sbom-report/SBOM_2022-12-14.json.
+
+This is an enterprise feature. Please visit https://debricked.com/pricing/ for more info.`,
```

Comment:
```suggestion
This is a premium feature. Please visit https://debricked.com/pricing/ for more info.`,
```
"premium feature" is used everywhere else in the CLI

Reply:
Premium and Enterprise are two different tiers with different features (see https://debricked.com/pricing/) SBOM export is only included in Enterprise (unlike other features currently available in the CLI)